manifests/w/w1hkj/flrig/2.0.10/w1hkj.flrig.locale.en-US.yaml

```
@@ -0,0 +1,13 @@
+# Created with komac v2.14.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.10.0.schema.json
+
+PackageIdentifier: w1hkj.flrig
+PackageVersion: 2.0.10
+PackageLocale: en-US
+Publisher: W1HKJ Software

```

```suggestion
Publisher: FLRIG developers
```
This should match what is in control panel or `AppsAndFeaturesEntries` should be specified in the installer manifest